ui/Dockerfile

```diff
@@ -1,27 +1,13 @@
-ARG NGINX_IMAGE_VERSION=1.15.8
 ARG NODE_IMAGE_VERSION=10.16.0
 
-FROM node:${NODE_IMAGE_VERSION} AS build-step
+FROM node:${NODE_IMAGE_VERSION}
 
 USER node
-WORKDIR /home/node
 
-# These are ordered by 'likeliness of change'
-COPY config-overrides.js .babelrc /home/node/
-COPY public /home/node/public
+RUN mkdir -p ~/ui
 
-COPY package*.json /home/node/
-RUN npm ci
+WORKDIR /home/node/ui
 
-COPY src /home/node/src
-RUN npm run build
+COPY entrypoint.sh /entrypoint.sh
```

Comment:
Why do you need to copy the entrypoint? You're mounting it when you use it to build the UI.

Reply:
So that we can also use that Dockerfile to build the npm stuff regardless of the buildchain.